mode/search.go

```
@@ -64,10 +65,7 @@ func (m SearchMode) Draw() {
 
 // Store the search term on the editor instance.
 // This allows us to use it later in other commands.
-func storeSearchTerm(e *editor.Editor, command string) error {
-	fields := strings.Fields(command)
-	term := fields[0]
-
+func storeSearchTerm(e *editor.Editor, term string) error {
```

This doesn't need to return an error